rootfs/etc/profile.d/prompt.sh

```diff
@@ -58,7 +68,7 @@ function geodesic_prompt() {
     ROLE_PROMPT="(none)"
   fi
 
-  PS1="\["$'${STATUS}'"\]"
+  PS1=$'${STATUS}'
```

Comment:
The extra brackets should only be used when ANSI is present. Otherwise, bash cursor position is miscalculated, in this case by 4 characters when pressing up-arrow to cycle through bash history.

![image](https://user-images.githubusercontent.com/52489/43626465-c8450cc8-96a6-11e8-8add-208afc44dd36.png)